nebula-logger/main/logger-engine/classes/LogEntryEventBuilder.cls

```diff
@@ -142,11 +186,7 @@ global without sharing class LogEntryEventBuilder {
         List<String> stackTraceLines = new List<String>();
         for (String currentStackTraceLine : stackTraceString.split('\n')) {
             // Don't include the logging system's classes in the stack trace
-            List<String> ignoredClasses = new List<String>{
-                Logger.class.getName(),
-                LogMessage.class.getName(),
-                LogEntryEventBuilder.class.getName()
-            };
+            List<String> ignoredClasses = new List<String>{ Logger.class.getName(), LogMessage.class.getName(), LogEntryEventBuilder.class.getName() };
```

Comment:
I would initialize this out of the for loop to save space/time

Reply:
Also since the list is being initialized simply to use `contains`, I believe a set would be more performant in general

Reply:
Good call - I'll make both of those changes